### DISCUSSION TITLE
Project Submission Button is missing

### DISCUSSION THREAD

**Discussion by surajbailwad** [github/india#824] [Students]
@ey-mentor1 , @ey-mentor2 , @EY-Mentor3 , @EY-Mentor4 "Save and Submit" button is replaced by "Deadline is exceed" in project tab even though project deadline is 30 march. Please resolve the issue.

Thank You and Regards.

**Comment by ey-mentor1** [comment]
This scenario usually comes when your project is under evaluation.
Can you please check now.

**Reply by surajbailwad** [reply]
@ey-mentor1  I had contacted the support they are saying I submitted 3 times. But I have submitted it only once. While I was about to submit the second time it started showing "project deadline exceed".

**Comment by surajbailwad** [comment]
Dear all, 
I have checked the Future Ready talent page it is still showing the same that is “Project deadline is exceed”
And it is also showing the status of submission as “Project not submitted”
Please resolve the issue

Thank you & Regards,
Suraj Bailwad
